libs/langchain/langchain/llms/symblai_nebula.py

```diff
@@ -68,33 +82,39 @@ def validate_environment(cls, values: Dict) -> Dict:
             DEFAULT_NEBULA_SERVICE_PATH,
         )
         nebula_api_key = get_from_dict_or_env(
-            values, "nebula_api_key", "NEBULA_SERVICE_API_KEY", ""
+            values, "nebula_api_key", "NEBULA_API_KEY", None
```

Comment:
believe this will still error if given a default value of None

Reply:
so you'd never hit the error message in L88

Reply:
@baskaryan you're right. Will remove the line 88 check. I tested it, the error gets reported if the value is set to None.

Reply:
@baskaryan its updated.